rst2ctags.py

```diff
@@ -54,9 +54,10 @@ def _formatFields(self):
         return '\t'.join(formattedFields)
 
     def __str__(self):
-        return '%s\t%s\t%s;"\t%s' % (
-                self.tagName, self.tagFile, self.tagAddress,
-                self._formatFields())
+        tag = '%s\t%s\t%s;"\t%s' % (
+            self.tagName, self.tagFile, self.tagAddress,
+            self._formatFields())
+        return tag.encode('utf-8')
```

Comment:
This is not enough.  If the string is raw, this will likely fail, unless it's all plain ascii characters.  It should be something more along the lines of:

```python
       if isinstance(tag, unicode):
           return tag.encode('utf-8')
       else:
           return tag
```